pkg/agent/util/syscall/syscall_windows.go

```diff
@@ -0,0 +1,367 @@
+//Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package syscall
+
+import (
+	"net"
+	"net/netip"
+	"os"
+	"strconv"
+	"syscall"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+)
+
+const (
+	AF_UNSPEC uint16 = uint16(windows.AF_UNSPEC)
+	AF_INET   uint16 = uint16(windows.AF_INET)
+	AF_INET6  uint16 = uint16(windows.AF_INET6)
+)
+
+// Declared variables which are meant to be overridden for testing.
+var syscallN = syscall.SyscallN
+
+type RouterDiscoveryBehavior int32
```

Comment:
Could we add some comments for types and constants for readers to understand what they are for / how they are used and where they are from (if we copy the definition from Windows APIs, documentation, standards, etc.)?